src/styles/index.scss

```diff
@@ -1,3 +1,84 @@
+$gap: 1px;
+$sizeSquare: 100px;
+$padding: 10px;
+$widthCalendar: $sizeSquare * 7 + $gap * 7 + $padding * 2;
+$days:
+  'mon' 0,
+  'tue' $sizeSquare + $gap,
+  'wen' 2 * ($sizeSquare + $gap),
+  'thu' 3 * ($sizeSquare + $gap),
+  'fri' 4 * ($sizeSquare + $gap),
+  'sat' 5 * ($sizeSquare + $gap),
+  'sun' 6 * ($sizeSquare + $gap);
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
 }
+
+.container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  width: $widthCalendar;
+  padding: $padding;
+
+  &--first-day-sun {
+    @each $day, $margin in $days {
```

Comment:
The `@each` loop is not correctly applying the margin-left for the first day of the week. The loop iterates over all days, but the `:first-child` selector will only apply to the first child of the calendar, not each day. Consider using a more specific selector or logic to apply the correct margin.